tests/Unit/ApparentHorizons/StrahlkorperGrTestHelpers.hpp

```diff
@@ -0,0 +1,26 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// Defines functions useful for testing StrahlkorperGr.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+/*!
```

Comment:
Let's expand the code in this file to include other things defined on Schwarzschild that are needed as inputs to the functions you are testing.  Put these functions in namespaces so you can shorten their names
```
namespace TestHelpers {
namespace Schwarzschild {
    ... spatial_ricci(...)
    ... unit_normal_one_form(...)
}
}
```

Reply:
After discussing on slack with @kidder, we decided to not make this change for now. I'm not sure how much good making this change will do, since most ingredients for the tests are already computed in one or two lines. If there's a consensus that it's worth the time, we can revisit moving more stuff into the helpers file.